lib/gringotts/gateways/securion_pay.ex

```diff
@@ -94,6 +94,25 @@ defmodule Gringotts.Gateways.SecurionPay do
     commit(params, "charges", opts)
   end
 
+  @doc """
```

Comment:
What happens to authorizations that are not captured (for a long time), do they expire?

Reply:
Sir, they are removed after the authorization expires ( 5 days).

Reply:
Please document this in `authorize/3` docs.

Reply:
Okay Sir